addons/web_editor/static/src/components/media_dialog/file_selector.xml

```diff
@@ -2,7 +2,7 @@
 <templates id="template" xml:space="preserve">
 <t t-name="web_editor.FileSelectorControlPanel" owl="1">
     <div class="d-flex flex-wrap gap-2 mt-4 mb-2 align-items-end">
-        <SearchMedia searchPlaceholder="props.searchPlaceholder" needle="props.needle" search="props.search"/>
+        <SearchMedia searchPlaceholder="props.searchPlaceholder" search="props.search"/>
```

Comment:
For the sake of stability, maybe do not remove the needle props in both file_selector and icon_selector XML? I guess removing `t-att-value="props.needle"` in the sub-component is inevitable though but probably less important since in JS and probably cannot be extended anyway :grimacing: 

(just add a comment that it will be removed in master)

Reply:
Ok, just removing the "props.needle" from the SearchMedia template should be enough to avoid the rerender via this prop, I will test that